modules/administration-guide/pages/adding-labels-and-annotations.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY
+:description: Labels and annotations
```

Comment:
Adding or configuring:
```suggestion
:description: Adding labels and annotations
```

Reply:
@tolusha, WDYT? I haven't read the content. This is the assembly title that includes both Kubernetes and OpenShift content in the upstream docs, and only OpenShift content in the downstream docs.